### PR TITLE
Use Absolute imports

### PR DESCRIPTION
**Pull request by markus-hinsche**
**Problem:** 
* Imports constantly confuse contributors
* `sys.path.append("/my/short/path")` is used in a lot of places but not a good solution

**Analysis** A lot of projects in the data science field (Tensorflow, Rasa, ...) use absolute imports, some (scikit-learn) use relative imports

**Solution:** Use absolute imports
* We have the special case of running parts of the codebase when we make a run on the cluster (e.g. train.py, evaluate.py) -> Here the new strategy is to pretend the directory of train.py is a module in the project root and you there is also a package `common` in this project root. 

Global directory structure:
```
├── src
│   ├── __init__.py
│   ├── analyses
│   ├── common
│   ├── data_utils
│   └── models
```

AzureML online Run directory structure:
```
├── common
│   ├── __init__.py
│   ├── cgmcore
│   ├── data_utilities
│   ├── depthmap_toolkit
│   ├── endpoints
│   ├── evaluation
│   ├── model_utils
│   ├── result_generation
│   ├── rgbd_dataset
│   ├── rgbd_toolkit
│   └── zscore
├── evaluate.py
```

**Changes:**
* Change relative import and import hacks to absolute imports
* Rename temporary directories `temp_common` to `common`
* remove check if offline and don't use copy_dir locally (only on remote/cluster run)
* update README to update contributor setup to set PYTHONPATH

**Checked:**
* [x] VS Code understand new absolute paths (You can put `PYTHONPATH=./src:${PYTHONPATH}` into the `.env` file)
- [x] eval pipeline still works ([Run](https://dev.azure.com/cgmorg/ChildGrowthMonitor/_build/results?buildId=4093&view=logs&j=3d42b299-58dd-525f-2f90-47c9dbee9ebd&t=3d42b299-58dd-525f-2f90-47c9dbee9ebd))
- [x] teach console by setting the PYTHONPATH
- [x] train.py still works locally
- [x] train.py still works on the cluster
- [x] zscore still works
- [x] local test setup works

**Future work**
* consider renaming `cgm-ml/src` to `cgm-ml/cgmml`

**Resources:**
- https://stackoverflow.com/questions/16981921/relative-imports-in-python-3

### CODE REVIEW

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/config.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 CONFIG = Bunch(dict(
     DATASET_MODE=DATASET_MODE_DOWNLOAD,
-    DATASET_NAME="anon-depthmap-95k",
```
Comment (prajurock) [inline src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/config.py:8]: rename back to 95k

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/train.py [hunk 2]
```diff
@@ -25,16 +24,10 @@
...
 
-if run.id.startswith("OfflineRun"):
-    # Copy common into the temp folder
-    common_dir_path = REPO_DIR / "src/common"
-    temp_common_dir = Path(__file__).parent / "temp_common"
-    copy_dir(src=common_dir_path, tgt=temp_common_dir, glob_pattern='*/*.py', should_touch_init=True)
-
-from temp_common.model_utils.model_plaincnn import create_cnn  # noqa: E402
-from temp_common.model_utils.preprocessing import (  # noqa: E402
+from common.model_utils.model_plaincnn import create_cnn  # noqa: E402
```
Comment (prajurock) [inline src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/train.py:27]: This is just renaming? Why we were copying this?

**Review by prajurock — APPROVED** [review]
Do we only have 2 occurrences of this relative import?

**Comment by prajurock** [comment]
z score working fine so marked it ticked.

**Comment by markus-hinsche** [comment]
> Do we only have 2 occurrences of this relative import?

I added this in more places now where we did relative imports.

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/train.py (hunk 2 above)
Comment (markus-hinsche) [reply src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height/src/train.py]: We were using copy_dir in order to access temp_common. 
Now we renamed it to `common` which can be found locally so copying is not required anymore. 

Note that for Online Runs (on the cluster) copying will still be done

**Review by prajurock — APPROVED** [review]
A lot of work!! Very important changes. I don't like sys too.

**Pull request merged**